taskcluster/ci/module-build/kind.yml

```diff
@@ -42,7 +42,6 @@ job-defaults:
       # add `win32-x86-64-gnu` back in to the list of targets here.
       - [bash, '-c', 'echo "rust.targets=arm,arm64,x86_64,x86,darwin,linux-x86-64\n" > local.properties']
     gradlew:
-      - ':{module_name}:testDebug'
```

Comment:
Ordinarily I would be a little concerned about not testing the actual code that will actually end up in the release artifacts, but we already weren't doing that here - `testDebug` will do a separate debug build of the android code in order to run the tests, and will run it on top of a separately-compiled megazord lib that then gets thrown away. So I don't see any downside to splitting this out into a separate task in this case.